packages/ckeditor5-image/src/imageresize/imageresizeediting.js

```diff
@@ -26,6 +26,37 @@ export default class ImageResizeEditing extends Plugin {
 		return 'ImageResizeEditing';
 	}
 
+	/**
+	 * @inheritDoc
+	 */
+	constructor( editor ) {
+		super( editor );
+
+		editor.config.define( 'image', {
```

Comment:
WDYT about moving this to `ImageResizeButtons` - the configuration options are defined there (the docs) and is only used by this plugin.

Reply:
So default unit stays in editing and `resizeOptions` go to `ImageResizeButtons`? Sounds like a plan.

Reply:
Hmm... well :man\_shrugging: for me TBH. Different sub-features defines different configuration options. But since it's already messy we can leave that as is.